### PR TITLE
Win Dev Environment instructions

### PR DESCRIPTION
**Pull request by HolySmoke86**
adds a rough outline of how to set up a dev env on windows

### CODE REVIEW

**Comment by HolySmoke86** [comment]
ah shoot, that should've been a [ci_skip]

**Comment by HolySmoke86** [comment]
@travis-ci uh, wat?

File: CONTRIBUTING.asciidoc [hunk 1]
```diff
@@ -526,6 +526,21 @@ generate code and subsequently overwrite part or all of it. Running with all
 will slow Valgrind down noticeably.
 ____
 
+Setting up a Windows Development Environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Install https://www.python.org/ftp/python/3.4.4/python-3.4.4.msi[Python 3.4] (32bit!)
```
Comment (The-Compiler) [inline CONTRIBUTING.asciidoc]: Why 32bit? Installing both 64bit should work fine, no?

File: CONTRIBUTING.asciidoc [hunk 2]
```diff
@@ -526,6 +526,21 @@ generate code and subsequently overwrite part or all of it. Running with all
...
+* Install https://www.qutebrowser.org/pyqt/PyQt5-5.5.1-gpl-Py3.4-Qt5.5.1-x32.exe[PyQt 5.5]
```
Comment (The-Compiler) [inline CONTRIBUTING.asciidoc]: I'd prefer a link to PyQt's sourceforge package, I mainly have those mirrored to make the CI builds more reliable, but I think we should point people to the upstream download.

File: CONTRIBUTING.asciidoc [hunk 3]
```diff
@@ -526,6 +526,21 @@ generate code and subsequently overwrite part or all of it. Running with all
...
+* Create a file at `C:\Windows\system32\python3.bat` with the following content:
+  `@C:\Python34\python %*`
+  This will make the Python 3.4 interpreter available as `python3`, which is used by various development scripts.
```
Comment (The-Compiler) [inline CONTRIBUTING.asciidoc]: Where is it actually still used? I don't really remember :laughing:

File: CONTRIBUTING.asciidoc [hunk 4]
```diff
@@ -526,6 +526,21 @@ generate code and subsequently overwrite part or all of it. Running with all
...
+* Install git from the https://git-scm.com/download/win[git-scm downloads page]
+  Try not to enable `core.autocrlf`, since that will cause flake to complain a lot. Use an editor that can deal with plain line feeds instead.
```
Comment (The-Compiler) [inline CONTRIBUTING.asciidoc]: flake -> flake8?

**Comment by The-Compiler** [comment]
Ugh.. some tests have race conditions, I don't think I've seen that one before though!

File: CONTRIBUTING.asciidoc (hunk 1 above)
Comment (The-Compiler) [inline CONTRIBUTING.asciidoc]: Do you need sip though?

Comment (HolySmoke86) [inline CONTRIBUTING.asciidoc]: Huh, I just removed all python slots on my windows machine and installed only python 3.4 and pyqt 5.5 for 64bit and now it works. I'll change the links to the respective download pages that contain both x86 and x86_64 installers as well as source, that's probably better, anyway

File: CONTRIBUTING.asciidoc (hunk 3 above)
Comment (HolySmoke86) [inline CONTRIBUTING.asciidoc]: `tox`, actually. otherwise gives an `InterpreterNotFound`

**Comment by HolySmoke86** [comment]
Alrighty, I'm much happier with this version (within the happiness limits set by python on windows)

**Comment by The-Compiler** [comment]
And so am I - thanks!

**Pull request merged**